pkg/standalone/standalone.go

```diff
@@ -396,12 +401,13 @@ func createComponentsDir(wg *sync.WaitGroup, errorChan chan<- error, dir, versio
 	componentsDir := getDefaultComponentsFolder()
 	_, err := os.Stat(componentsDir)
 	if os.IsNotExist(err) {
-		errDir := os.MkdirAll(componentsDir, 0755)
+		errDir := os.MkdirAll(componentsDir, 0777)
 		if errDir != nil {
 			errorChan <- fmt.Errorf("error creating default components folder: %s", errDir)
 			return
```

Comment:
The correct fix as suggested by @LMWF is to move copying of the default yaml files to dapr init. But the code looks too involved (package dependencies with components loader) for a last minute fix. I can address that in the next release